jupyter/nmm-game-utils.ipynb

```diff
@@ -322,7 +322,23 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Mills"
+    "## Hilfsfunktionen für Mühlen\n",
+    "In diesem Kapitel werden Hilfsfunktionen implementiert, die für Mühlen nützlich sind."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Die Funktion `findMills` berechnet für ein Spielbrett und einen Spieler alle Mühlen, die dieser Spieler aktuell hat. Die Funktion hat zwei Argumente:\n",
+    "\n",
+    "* `board` $\\in \\{s[1] | s \\in States\\}$ ist ein Spielbrett\n",
```

Comment:
müssen wir es wirklich so überverkomplizieren? Gefällt mir persönlich nicht und verwirrt meiner Meinung nach eher

Reply:
Also es gibt noch eine schönere Schreibweise, die mir gerade erst eingefallen ist:
`board € {b | (stash, b) in States}`

Reply:
* `board` ist ein Spielbrett;

Reply:
@nclskfm hat den Streit geklärt ;)

Reply:
Danke @nclskfm !